governance-aepp/src/views/Create.vue

```diff
@@ -94,8 +94,7 @@
 <script>
 
 
-  import pollIrisContractSource from '../assets/contracts/Poll_Iris.aes';
-  import BiggerLoader from '../components/BiggerLoader';
+import BiggerLoader from '../components/BiggerLoader';
```

Comment:
is this a formatting thing? indentation seems to be different

Reply:
fixed